airbyte-integrations/connectors/source-hubspot/source_hubspot/streams.py

```diff
@@ -1264,7 +1302,7 @@ class DealPipelines(ClientSideIncrementalStream):
     created_at_field = "createdAt"
     cursor_field_datetime_format = "x"
     primary_key = "pipelineId"
-    scopes = {"contacts", "tickets"}
+    scopes = {"contacts", "tickets", "crm.objects.deals.read"}
```

Comment:
According to docs we don't need deals scopes, only contacts for deals pipeline or tickets - https://legacydocs.hubspot.com/docs/methods/pipelines/get_pipelines_for_object_type